repositories/device.go

```diff
@@ -75,11 +80,32 @@ func (m *DeviceModel) fromModel() twomes.Device {
 }
 
 func (r *DeviceRepository) Find(device twomes.Device) (twomes.Device, error) {
+	fmt.Println("Find device", device.ID)
 	deviceModel := MakeDeviceModel(device)
-	err := r.db.Preload("DeviceType").Preload("Uploads").Where(&deviceModel).First(&deviceModel).Error
+	err := r.db.Preload("DeviceType").Preload("Uploads").Preload("Uploads.Measurements").Where(&deviceModel).First(&deviceModel).Error
```

Comment:
`.Preload("Uploads")` can be removed, since Uploads.Measurements also covers preloading uploads.

Reply:
Fixed